docs/content/en/docs/getting-started/production-environment/_index.md

```diff
@@ -196,6 +196,10 @@ Follow these steps if you want to use your initial cluster to create and manage
 
    As noted earlier, adding the `--kubeconfig` option tells `eksctl` to use the management cluster identified by that kubeconfig file to create a different workload cluster.
 
+   {{% alert title="Note" color="primary" %}}
```

Comment:
This looks like it is not showing up correctly on preview.

Reply:
That is a Hugo directive, is there a preview for that other than manually running this. Sometimes I throw a screen cap on the PR for these